inclusive-design/idrc
# fix: use project-specific Markdown filter (resolves #634)

## Description

With one exception, this PR resolves the issue identified in #634.

## Steps to test

1. Run `npm start`.

**Expected behavior:** Only one instance of the console message appears as opposed to a couple dozen before. 

## Additional information

Not applicable.

## Related issues

Resolves #634.
